### PR TITLE
Whole Foods

### LINKED ISSUE #35: Whole Foods
http://www.wholefoodsmarket.com/stores/list

### PR DESCRIPTION
**Pull request by AnthoneyKalasho**
Fixes #35

### CODE REVIEW

**Comment by AnthoneyKalasho** [comment]
There's opening hours listed but the format is very irregular from store to store, I'm sure I could hack something together but I can't see it being very accurate. Also I'm not sure what to do without lat/long data

File: locations/spiders/whole_foods.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import scrapy
+from locations.items import GeojsonPointItem
+import re
+import string
+
+class WholeFoodsSpider(scrapy.Spider):
+    name = "whole_foods"
+    allowed_domains = ["www.wholefoodsmarket.com"]
+    start_urls = (
+        'https://www.wholefoodsmarket.com/stores/list?page=0',
```
Comment (iandees) [inline locations/spiders/whole_foods.py]: Looks like you can get lat/lon by starting at `https://www.wholefoodsmarket.com/ajax/stores`, then getting more info by spidering to "https://www.wholefoodsmarket.com/stores/<path>" where `<path>` is in the JSON response.

File: locations/spiders/whole_foods.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    def parse_phone(self, phone):
+        phone = phone.replace('.','')
+        phone = phone.replace(')','')
+        phone = phone.replace('(','')
+        phone = phone.replace('_','')
+        phone = phone.replace('-','')
+        phone = phone.replace('+','')
+        phone = phone.replace(' ','')
+        phone = phone.replace('','')
```
Comment (iandees) [inline locations/spiders/whole_foods.py]: I don't think you need this extra `replace()`.

File: locations/spiders/whole_foods.py (hunk 1 above)
Comment (AnthoneyKalasho) [reply locations/spiders/whole_foods.py]: I'll rewrite the spider tomorrow using this

**Pull request merged**